docs/config/templates/app/indexPage.template.html

```diff
@@ -146,7 +146,8 @@ <h4 class="search-results-group-heading">{{ key }}</h4>
         <div class="navbar-inner">
           <div class="container">
             <p class="site-notice">
-              AngularJS is now in Long Term Support (LTS) mode: <a href="https://docs.angularjs.org/misc/version-support-status">Find out more</a>. Explore End Of Life (EOL) options <a href="https://blog.angular.io/finding-a-path-forward-with-angularjs-7e186fdd4429">here</a>.
+              <a href="https://goo.gle/angularjs-end-of-life">Support for legacy AngularJS has fully ended.</a>
+              Visit <a href="https://angular.io">https://angular.io</a> for the actively supported Angular.
```

Comment:
The reason I put this here is because there's some subset of people that this part of the page and jump to the incorrect conclusion that this is referring to Angular. I'd like to keep this message everywhere to prevent that.